src/store/store.ts

```diff
@@ -100,7 +101,7 @@ export const useStore = defineStore('store', {
 
     /** Whether the current filing is an Amalgamation Application. */
     isAmalgamationFiling (): boolean {
-      return (this.stateModel.tombstone.filingType === FilingTypes.AMALGAMATION_APPLICATION)
+      return (this.getFilingType === FilingTypes.AMALGAMATION_APPLICATION)
```

Comment:
I prefer this because it means fewer detailed de-references of the stateModel object, and I think the value is cached this way.

Also, if we ever want to change where the data is stored, we only have to change 1 getter instead of many (and 1 action).